### ISSUE TITLE
FreeBSD-12.0-i386 on VMware Workstation: dhclient blocking 

### ISSUE THREAD

**Issue by arizvisa** [chef/bento#1263]
For some reason on my system, building the `FreeBSD-12.0-i386` template blocks when trying to fetch the ip address via `dhclient`. This happens when `packer` is typing in the boot sequence prior to it waiting for ssh to come up.

This will look like the following in workstation's console:
```
...
# mdmfs -s 100m md1 /tmp
random: unblocking device.
# mdmfs -s 100m md2 /mnt
# dhclient -l /tmp/dhclient.lease.em0 em0
Cannot open or create pidfile: Read-only file system
```

For some reason however this is only happening to me with `FreeBSD-12.0-i386` and \_not\_ `FreeBSD-12.0-amd64` which honestly shouldn't be the same codebase and shouldn't result in different symptoms. Nonetheless, I confirmed by trying on multiple VMware hosts all with the same result of `dhclient` blocking after displaying that error, and not backgrounding itself so the next `fetch` command can download the script for `bsdinstall`.

**Comment by arizvisa** [comment]
To fix this, I simply updated the `boot_command` in each of the builders for the template and added the `-p` parameter to tell `dhclient` to write its pidfile to `/tmp`.

So where you see this line under `boot_command` (one of them uses `vtnet0`):
```
"dhclient -l /tmp/dhclient.lease.em0 em0<enter><wait>",
```

You can change it to the following. By default `dhclient` will try to write it to `/var/run/dhclient.$interface.pid`.
```
"dhclient -p /tmp/dhclient.em0.pid -l /tmp/dhclient.lease.em0 em0<enter><wait>",
```

**Comment by tas50** [comment]
@arizvisa Can you submit a PR for that?

**Comment by arizvisa** [comment]
Yeah, sure. I didn't write one up at first because I'm still not sure if I'm crazy because there's literally no logic change between the i386 and amd64 binaries for `dhclient`.

Should I update all of the FreeBSD templates to specify the path for the pidfile as a defensive measure?

**Comment by arizvisa** [comment]
Ok. PR #1264 fixes this for me.

**Comment by arizvisa** [comment]
Closed by PR #1264.
